lib/src/ui/common/error/error_message_provider.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:allthenews/generated/l10n.dart';
+import 'package:allthenews/src/data/communication/api/api_exception.dart';
+import 'package:flutter/cupertino.dart';
+
+class ErrorMessageProvider {
+  static String provideErrorMessage(Exception exception, BuildContext context) =>
+      provideErrorKey(exception, context);
+
+  static String provideErrorKey(Exception exception, context) {
```

Comment:
też chyba nie ma sensu wydzielenie tej metody skoro to jest jedyne co wywołuje provideErrorMessage